Frontend/mobile/app/auth/register.tsx

```
@@ -328,39 +282,46 @@ export default function RegisterScreen() {
+    </Text>
+  )}
+</TouchableOpacity>
+
 
         <TouchableOpacity 
           onPress={() => !isLoading && router.push('./login')}
           disabled={isLoading}
         >
-          <Text style={[styles.link, isLoading && styles.linkDisabled]}>
+          <Text style={[styles.link, , { color: themeStyle.text }, isLoading && styles.linkDisabled]}>
             Already have an account? Login
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)